server/migrations/2023-02-14-193708_initial_schema/up.sql

```diff
@@ -1,8 +1,11 @@
 -- Your SQL goes here
 
 CREATE TABLE mina_proposals (
-  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
-  key TEXT NOT NULL,
+  id SERIAL PRIMARY KEY,
+  key TEXT NOT NULL UNIQUE,
   global_start_slot INTEGER NOT NULL,
-  global_end_slot INTEGER NOT NULL
-);
\ No newline at end of file
+  global_end_slot INTEGER NOT NULL,
+  ledger_hash TEXT
+);
+
+INSERT INTO mina_proposals VALUES (1, 'MIP1', 316140, 320791, 'jxQXzUkst2L9Ma9g9YQ3kfpgB5v5Znr1vrYb1mupakc5y7T89H8'); 
```

Comment:
Once we've deployed this version, we're going to have to use new migrations when making schema changes. It also likely needs an index on key since that's how we'll be querying the proposals